pex/interpreter.py

```diff
@@ -152,7 +154,7 @@ def get(cls, binary=None):
     def decode(cls, encoded):
         TRACER.log("creating PythonIdentity from encoded: %s" % encoded, V=9)
         values = json.loads(encoded)
-        if len(values) != 11:
+        if len(values) != 12:
```

Comment:
It would take some fancy introspection of the fields -> field count of PythonIdentity. That would be easy it it were `@attr.s`'d, but its not yet. If PythonIdentity / PythonInterpreter can be `@attr.s`'d without harming PEX bootstrap time perf, this would be a good idea. I'm not sure about the perf aspect though - I assume that's why I have not done this yet.